roles/etcd/defaults/main.yaml

```diff
@@ -0,0 +1,8 @@
+---
+etcd_client_port: 4001
+etcd_peer_port: 7001
+etcd_addr: "{{ ansible_fqdn}}:{{ etcd_client_port }}"
+etcd_peer_addr: "{{ ansible_fqdn}}:{{ etcd_peer_port }}"
```

Comment:
After my recent changes land, you'll be able to use either openshift_hostname or openshift_public_hostname (depending on whether we want to have it externally available or not). openshift_hostname uses the ansible_nodename fact if not explicitly set (though I plan to update it to use hostname -f eventually), and openshift_public_hostname uses ansible_fqdn by default if not set (which is determined by doing ip resolution on the default ip address of the host).